### DISCUSSION TITLE
2023-06-29

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#1] [daily_meeting]
# 오늘 할 일
## 개인 일정
노아 - 5시 근로 회의

## 제출할 것

1. 프로젝트 한 줄 요약
2. 킥오프 미팅: 예상 기능 목록을 적기 위해 브레인스토밍을 해야 함.

## 추가적인 것

1. 1차 데모까지의 목표 설정
2. 사용자스토리 관련 마무리
3. 회의록 등을 위한 문서 기록 가이드 (그라운드 룰에 포함, 아마 내일)

## 분야 별

1. 기술 스택 정의
2. 컨벤션 정의

# 내일 할 일
- 그라운드 룰 정하기
- Github milestone 설정
- 문서 작성 (이슈, 디스커션)템플릿 및 작성 가이드 문서화
- 프론트 백엔드에 대한 코드 컨벤션 문서화

**Comment by jaehee329** [comment]
### 체크인
7점 - 방학때 쉰 에너지 다 쓴 것 같아요. 길게 잤는데도 피곤하네요
### 체크아웃
5점 - 피곤해졌어요
### KPT
K: 어제보다 정해진 주제에 대해서 논의가 수월하게 진행됐어요
P: 함께 추가적으로 논의해야 할 사항들이 너무 많다. 요구사항과 기능에 대한 구체화
T: 이번 스프린트까지의 목표를 명확히 하고 마일스톤들을 내일 설정해보면 어떨까? 일정 추정도 바로 이어질 수 있을 것 같다.

**Comment by woosung1223** [comment]
### 체크인
6점 - 피곤한데(-2) 비와서(-2) 6점입니다..!

### 체크아웃
8점 - 오늘 해야 할 일을 어느정도 끝낸 것 같아 기분이 좋습니다.

### KPT
K: 이야기가 산으로 가는 빈도가 어제보다 줄었다는 것.
P: 의견 제시, 조율 시 다른 크루들과 부드럽게 논의를 이어나갔으면 어땠을까 싶다!
T: 회의를 할 때, 어떤 점이 좋았는지 중간중간에 칭찬을 섞어보면 어떨까요? (ex. 그런 아이디어 되게 좋았어!)

**Comment by MoonJeWoong** [comment]
### 체크인 
(9점 / 10점) - 이제 개강하고 3일차인데 생활 주기가 다시 바로 잡혀가는 것 같아서 오늘 컨디션이 괜찮습니다. 원래는 10점을 줄만 한데 날씨가 습해서 1점 깎았습니당.


### 체크아웃 
(8점 / 10점) - 우리가 만들어야 할 서비스의 큰 그림이 이제 그려져 가는 것 같아서 즐거웠습니다. 다만 하루종일 회의하면서 피곤해져서 2점 깎았습니다.

### KPT

K: 회의가 산으로 가거나 길게 늘어지는 것을 방지하기 위해 회의 시간과 쉬는 시간을 명확하게 정해놓고 진행했던 점이 좋았습니다.
P: 생각했던 것보다 해야할 일을 많이 하지 못한 점이 아쉽습니다.
T: 우리가 논의하는 내용이 오늘 해야할 일을 해결하는데 필수적인 내용인지 확인하자.

**Comment by nlom0218** [comment]
### 체크인

9점 - 어제 도시락통을 까먹고 가져가지 않아 찜찜해서 1점 깍임

### 체크아웃

10점 - 오늘은 도시락을 꼭 가져가야겠다. 팀명 만족! 팀 한 줄 소개 만족!

### KPT

K: 회의 공간이 아닌 넓은 공간(무중력 광장)에서 이야기를 나누니 색다른 느낌을 받음. 긴 내용이 아니라 짧은 내용을 다룰 때 효과적이라고 생각함. 의도에 벗어난 이야기를 어제보다 하지 않음
P: 서로 생각하는 방향이 맞지 않음. 이를 통일을 할 수 있는 방안이 있었으면 함.
T: 회의의 진행이 자연스럽고 흐름이 좋을 때에도 과연 50분 10분을 지켜야 하는지에 대해 이야기를 나누어봤으면 좋겠다.

**Comment by yeopto** [comment]
### 체크인
8점 - 어제 일찍 자서 개운했는데 지옥철 때문에 -2점

### 체크아웃
5점 - 역시 기획은 쉽지 않다. 회의로 인해 녹초가 됨.

### KPT

K : 회의 시간을 적당히 정하고 하다보니 좀 더 집중할 수 있었던 것 같다.
P : 방향성에 대해 서로 동기화가 덜 되어있었던게 아쉽다.
T : 회의 시작 전 주제에 대해 서로 생각하는 것이 같은지 확인하고 시작해봤으면 좋겠다.

**Comment by woo-jk** [comment]
### 체크인
8점 - 조금 피곤하구먼
### 체크아웃
9점 - 팀 기획이 날마다 빠르게 진전되는 것이 느껴져서 기분 좋음

### KPT
K: 회의 시간을 명확히 정해놓고 하니 회의의 흐름이 훨씬 좋았다. 그리고 회의실에서 화이트보드와 함께 진행하니 더 집중이 잘됐다. 중요한 회의는 회의실에서 직접 그려가며 하면 더 좋을 듯
P: 의견에 대해 좋았던 점을 칭찬하는 빈도보다 안좋은 점에 대해 반박하는 경우의 빈도가 더 많은 것 같다. 나도 그렇다. 이 문제 때문에 우리의 회의는 조금 딱딱한 느낌이 있다.
T: 서로의 좋은 의견에 대한 칭찬 빈도를 많이 늘려보자

**Comment by aak2075** [comment]
### 체크인
8점 - 조금 피곤하다
 
### 체크아웃
8점 - 어느새 일주일이 끝나가는 것이 좋다.

### KPT
K : 짚고 넘어가야할 사항을 확실하게 짚고 넘어가는 것이 좋았다.
P : 서로 생각하는것이 다를 때 이를 맞추는데 어려움이 있었다.
T : 서로 다른 생각을 빠르게 파악하고 맞추는 방법을 생각하면 좋을듯.
